sdk/tests/conformance/extensions/ext-sRGB.html

```diff
@@ -402,10 +406,27 @@
   }
 }
 
-debug("");
+function runLoadFromImageTest(callback) {
+  debug("");
+  debug("Tests to ensure that SRGB textures can successfully use image elements as their source");
+
+  var img = wtu.makeImage("../resources/grayscale-sun.jpg", function() {
+    var tex = gl.createTexture();
+    gl.bindTexture(gl.TEXTURE_2D, tex);
+    gl.texImage2D(gl.TEXTURE_2D, 0, ext.SRGB_EXT, ext.SRGB_EXT, gl.UNSIGNED_BYTE, img);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR);
+
+    gl.texImage2D(gl.TEXTURE_2D, 0, ext.SRGB_ALPHA_EXT, ext.SRGB_ALPHA_EXT, gl.UNSIGNED_BYTE, img);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR);
+
+    callback();
+  }, function() {
+    testFailed("Image could not be loaded");
+    callback();
+  });
+}
+
 var successfullyParsed = true;
```

Comment:
What I mean is you don't need "var successfullyParsed = true;" because finishTest() takes care of it.  Well, no big deal.